docs/integrations/dbt.md

```diff
@@ -1,19 +1,22 @@
 # dbt
 
-SQLMesh has native support for running dbt projects. This featuring is currently under development. You can view the development backlog [here](https://github.com/orgs/TobikoData/projects/1/views/3). If you are interested in this feature, we encourage you to try it with your dbt projects and submit issues here (https://github.com/TobikoData/sqlmesh/issues), so we can make it more robust.
+SQLMesh has native support for importing and running dbt projects. 
```

Comment:
We don't really import the project. We read the format and use as is. Import, to me, implies we convert it to our format and then you use that going forward.

Reply:
Me too - good call